pkg/grpc/proxy/handler_test.go

```diff
@@ -177,7 +185,10 @@ func (s *ProxyHappySuite) TestDirectorErrorIsPropagated() {
 }
 
 func (s *ProxyHappySuite) TestPingStream_FullDuplexWorks() {
-	stream, err := s.testClient.PingStream(s.ctx())
```

Comment:
Why this change? This reduces the timeout

Reply:
I removed the following method, and then used the `context. WithTimeout` method directly. It is not appropriate to put ctx in the `ProxyHappySuite` object, it should be placed in a entire request.

```golang
func (s *ProxyHappySuite) ctx() context.Context {
```